### DISCUSSION TITLE
Feign uses Okhttp connection pool, resulting in long display time for Skywalking UI

### DISCUSSION THREAD

**Discussion by xiaozichen** [apache/skywalking#12137] [Q&A]
skywalking-aop:8.9.1 ,
agent: 8.13.0

The use of feign and Okhttp connection pools in microservices leads to connection reuse, resulting in a long display time in the UI

![image](https://github.com/apache/skywalking/assets/24701842/47f9968f-a0f2-4a5f-8cda-4db21093bf0a)

Do you have a good solution？

**Comment by chen1147344634** [comment]
do you resolve？
